cuda_core/examples/saxpy.py

```diff
@@ -73,8 +74,8 @@
 # prepare input
 size = cp.uint64(128)
 a = dtype(42)
-x = cp.random.random(size, dtype=dtype)
-y = cp.random.random(size, dtype=dtype)
+x = rng.random(size, dtype=dtype)
```

Comment:
I'm not against having this in the review, but it feels like a separate change

Reply:
This change is noted in the PR description. Without this change, we cannot use CuPy in the test CI. See the previous failed attempt here:
https://github.com/NVIDIA/cuda-python/actions/runs/12365525056/job/34510924292#step:14:170